### PR TITLE
added few animation classes rotate, swirl, scale in and tilt in

### PR DESCRIPTION
**Pull request by bemijonathan**
Screenshot from 2020-05-26 14-42-41 - 1

### CODE REVIEW

File: src/tilt-in.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+/* tilt in animation */
+.tilt-in-enter,
+.tilt-in-leave-to{
+    transform: rotateY(20deg) rotateX(35deg) translate(300px, -300px) skew(-35deg, 10deg);
+    opacity: 0;
+}
+
+.tilt-in-enter-active,
+.tilt-in-leave-active{
+    transition:  transform 0.3s, opacity 0.5s ;
```
Comment (Edmund1645) [inline src/tilt-in.css:11]: The transform duration here is little as well, making the animation too fast, please.

also, try adding `transition-timing-function` to make the animation slow down a bit towards the end.

File: demo/report.20200526.101148.3406.0.001.json [hunk 2]
```diff
@@ -0,0 +1,552 @@
+
```
[the remark this answers is not in the document]
Comment (bemijonathan) [reply demo/report.20200526.101148.3406.0.001.json:1]: sure I didn't  notice this

File: src/swirl.css [hunk 3]
```diff
@@ -0,0 +1,11 @@
+
+  .swirl-enter,
```
[the remark this answers is not in the document]
Comment (bemijonathan) [reply src/swirl.css:2]: I changed the preview content to a box so the difference in the animations can be noticed
Screenshot from 2020-05-27 14-08-44

**Comment by Edmund1645** [comment]
@all-contributors please add @bemijonathan for code and maintenance

**Pull request merged**